lib/peep.rb

```diff
@@ -0,0 +1,27 @@
+require_relative './database_connection'
+
+class Peep
+  attr_reader :id, :message, :time
+
+  def initialize(id:, message:, time:)
+    @id = id
+    @message = message
+    @time = time
+  end
+
+  def self.all
+    result = DatabaseConnection.query("SELECT * FROM peeps;")
+    result.map { |peep|
+      Peep.new(id: peep['id'], message: peep['message'], time: peep['time'])
+    }
+  end
+
+  def self.post(message)
```

Comment:
This is a small point, but conventionally this method would be called create rather than post.